modules/mbedtls/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ if(CONFIG_MBEDTLS_BUILTIN)
   target_include_directories(mbedTLS INTERFACE
     ${ZEPHYR_CURRENT_MODULE_DIR}/include
     configs
+    mbedtls_init
```

Comment:
I think it could be just `include`, so that other headers would fit into that directory, like here: https://github.com/zephyrproject-rtos/zephyr/pull/46572/commits/30f07ed81f9f68d4c792ba0b062fc4d57a4a8589

Reply:
i shall go take a look thanks